src/gui/guiiconprovider.cpp

```diff
@@ -117,12 +129,13 @@ QIcon GuiIconProvider::generateDifferentSizes(const QIcon &icon)
 
     return newIcon;
 }
+
 #endif
 
-QString GuiIconProvider::getIconPath(const QString &iconId)
+QString GuiIconProvider::getIconPath(const QString &iconId, const QString &overrideTheme)
```

Comment:
This was after rebasing, and I didn't see any changes. Will become moot if @sledgehammer999 wants me to reorganise the icon hierarchy.